normalhighschools/november.md

```diff
@@ -52,12 +52,13 @@ Below are topics to be assigned to small groups in the classrooms. The small gro
   - Any other items of note to share from your research or further questions?
 
 
-### Laws and regs
+### Laws, regs, and safety
```

Comment:
Students may not understand shorthand of "regs". I suggest spelling out "regulations".